Dockerfile

```diff
@@ -41,7 +43,12 @@ RUN phpize && \
 
 WORKDIR /build
 
-RUN composer install && \
+RUN php -r "copy('https://getcomposer.org/installer', 'composer-setup.php');" && \
+    php -r "if (hash_file('SHA384', 'composer-setup.php') === '544e09ee996cdf60ece3804abc52599c22b1f40f4323403c44d44fdfdd586475ca9813a858088ffbc1f233e9b180f061') { echo 'Installer verified'; } else { echo 'Installer corrupt'; unlink('composer-setup.php'); } echo PHP_EOL;" && \
```

Comment:
The hash value occasionally changes.

In php-docker, it fetchs the signature on demand:
https://github.com/GoogleCloudPlatform/php-docker/blob/master/php-base/build-scripts/install_composer.sh#L21

It has been working well.

Reply:
Ok will adapt something like this to install composer.